services/core-api/app/api/parties/party/models/party.py

```diff
@@ -1,55 +1,60 @@
 from datetime import datetime
 import re
-import uuid
 
-from sqlalchemy import func
+from sqlalchemy import func, case, and_
 from sqlalchemy.schema import FetchedValue
 from sqlalchemy.dialects.postgresql import UUID
 from sqlalchemy.ext.hybrid import hybrid_property
 from sqlalchemy.orm import validates
-from app.extensions import db
 from werkzeug.exceptions import BadRequest
 
+from app.extensions import db
 from app.api.utils.models_mixins import SoftDeleteMixin, AuditMixin, Base
 from app.api.parties.party.models.address import Address
 
+MAX_NAME_LENGTH = 100
+
 
 class Party(SoftDeleteMixin, AuditMixin, Base):
     __tablename__ = 'party'
+
     party_guid = db.Column(UUID(as_uuid=True), primary_key=True, server_default=FetchedValue())
-    first_name = db.Column(db.String, nullable=True)
-    middle_name = db.Column(db.String, nullable=True)
+    first_name = db.Column(db.String)
+    # TODO: Determine if this column is necessary; it appears to not be used by anything.
```

Comment:
Can we delete this column? It's not being used anywhere.

Reply:
you want to delete "first_name"  or "middle_name"?

"middle_name" yes, I'd delete - not sure why we have that.